src/main/java/org/apache/commons/crypto/examples/RandomExample.java

```diff
@@ -32,29 +32,62 @@
 public class RandomExample {
 
     public static void main(String []args) throws GeneralSecurityException, IOException {
-        //Constructs a byte array to store random data.
-        byte[] key = new byte[16];
-        byte[] iv = new byte[32];
+        exampleWithdefaultHardwareRandom();
+        exampleWithUserDefinedRandom();
+    }
+
+    public static void exampleWithdefaultHardwareRandom() {
+        try {
+            //Constructs a byte array to store random data.
+            byte[] key = new byte[16];
+
+            //Gets the 'CryptoRandom' instance.
+            CryptoRandom random = CryptoRandomFactory.getCryptoRandom();
+
+            // Generate random bytes and places them into the byte arrays.
+            random.nextBytes(key);
+
+            //Closes the CryptoRandom.
+            random.close();
+
+            // Show the generated output
+            System.out.println(Arrays.toString(key));
+
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        }
```

Comment:
This is sample code so should be as simple as possible.
Drop the try/catch and throw Exception from the method instead